socorro/processor/mozilla_transform_rules.py

```diff
@@ -813,8 +813,8 @@ def __init__(self, config):
     def version(self):
         return '1.0'
 
-    def _get_version_data(self, product, version, release_channel, build_id):
-        key = '%s:%s:%s:%s' % (product, version, release_channel, build_id)
+    def _get_version_data(self, product, version, build_id):
+        key = '%s:%s:%s' % (product, version, build_id)
```

Comment:
I don't understand the change here. So I get that we're removing `release_channel` from the key for caching and we're removing it from the SQL, too. But what is `_get_version_data` returning and now do these changes affect whatever is calling it?